test/functional/details_screen_spec.js

```diff
@@ -54,8 +52,36 @@ describe('The User Details screen...', function() {
                 data: "Gata"
             },
             {
-                title: "Birth Date",
+                title: "Company Email",
+                data: "cutest.kitty@kiva.org"
+            },
+            {
+                title: "Hire Date",
                 data: "2019-06-17"
+            },
+            {
+                title: "Current Title",
+                data: "Kiva's Cutest Kitty"
+            },
+            {
+                title: "Team",
+                data: "Kiva Protocol"
+            },
+            {
+                title: "Office Location",
+                data: "San Francisco"
+            },
+            {
+                title: "Employment Type",
+                data: "Full Time"
+            },
+            {
+                title: "End Date",
+                data: "NEVER!"
```

Comment:
don't these inputs need to be date formats?

Reply:
They don't for the Details screen, because we're rendering the received data as-is (i.e. we're not doing any data parsing to confirm that the End Date is an actual date)